istioctl/cmd/authz.go

```diff
@@ -304,13 +373,13 @@ func init() {
 	checkCmd.PersistentFlags().BoolVarP(&printAll, "all", "a", false,
 		"Show additional information (e.g. SNI and ALPN)")
 	checkCmd.PersistentFlags().StringVarP(&configDumpFile, "file", "f", "",
-		"Check the Envoy config dump from a file")
-	convertCmd.PersistentFlags().StringSliceVarP(&policyFiles, "file", "f", []string{},
-		"v1alpha1 RBAC policy that needs to be converted to v1beta1 authorization policy")
+		"The json file with Envoy config dump to be checked")
+	convertCmd.PersistentFlags().StringSliceVarP(&v1Files, "file", "f", []string{},
+		"The yaml file with v1alpha1 RBAC policies to be converted")
 	convertCmd.PersistentFlags().StringSliceVarP(&serviceFiles, "service", "s", []string{},
-		"Kubernetes Service resource that provides the mapping between service and workload")
-	convertCmd.PersistentFlags().StringVarP(&meshConfig, "meshConfigFile", "m", "",
-		"Istio MeshConfig file that provides the root namespace value")
-	convertCmd.PersistentFlags().StringVar(&istioMeshConfigMapName, "meshConfigMapName", "istio",
-		"ConfigMap name for Istio mesh configuration")
+		"The yaml file with Kubernetes services for the mapping from the service name to workload selector, used with -f")
+	convertCmd.PersistentFlags().StringVarP(&rootNamespace, "rootNamespace", "r", "istio-system",
```

Comment:
I remember you didn't want users to provide `rootNamespace` directly before. Just curious, what changed?

Reply:
The primary use case is still to get the root namespace from the cluster and `--rootNamespace` is  only used with `-f` and I think in this case (offline mode), the user may not have the meshconfig or even Istio cluster at all.